imgoing-frontend/src/screens/AuthLoadingScreen.tsx

```diff
@@ -30,13 +32,15 @@ const AuthLoadingScreen = () => {
 
   useEffect(() => {
     if (accessToken === undefined) return;
-
     if (accessToken === null) {
       navigation.navigate('Login');
     } else {
-      // TODO : plan 조회
       dispatch(getBookmarkList());
-      navigation.navigate('Main');
+      (async () => {
+        const { data } = await request('plan');
+        dispatch(initPlan(data.data.map(ResToPlan)));
```

Comment:
```suggestion
        dispatch(getPlanList());
```

thunk를 사용하고 있어서 따로 data를 받을 필요가 없습니다!
thunks/plan.ts의 getPlanList 함수 보시면 반환된 값이 store로 넘어가서 저장됩니다!

Reply:
이렇게되면 데이터를 받기전에 넘어가 버려서 HomeScreen에서 HomeLanding이 먼저 보이게되더라구요
그래서 어떻게 해야할지..